### PR TITLE
Add HealthCheck basic implementation

### PR DESCRIPTION
**Pull request by Kilometerr**

### CODE REVIEW

File: DiscordBot/DiscordBotCore/src/main/java/com/tainavbot/server/discordbot/domain/enumeration/HealthStatusEnum.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.tainavbot.server.discordbot.domain.enumeration;
+
+public enum HealthStatusEnum {
+    OK,
+    DEAD
```
Comment (Maje-afk) [inline DiscordBot/DiscordBotCore/src/main/java/com/tainavbot/server/discordbot/domain/enumeration/HealthStatusEnum.java]: DEAD to NOT_OK

File: Orchestrator/src/main/java/com/tainavbot/server/orchestrator/domain/enumeration/HealthStatusEnum.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.tainavbot.server.orchestrator.domain.enumeration;
+
+public enum HealthStatusEnum {
+    OK,
+    DEAD
+}
```
Comment (Maje-afk) [inline Orchestrator/src/main/java/com/tainavbot/server/orchestrator/domain/enumeration/HealthStatusEnum.java]: Duplicate

File: Orchestrator/src/main/java/com/tainavbot/server/orchestrator/infrastructure/adapter/primary/api/dto/enumeration/HealthStatusEnumResponse.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.tainavbot.server.orchestrator.infrastructure.adapter.primary.api.dto.enumeration;
+
+public enum HealthStatusEnumResponse {
+    OK, DEAD
+}
```
Comment (Maje-afk) [inline Orchestrator/src/main/java/com/tainavbot/server/orchestrator/infrastructure/adapter/primary/api/dto/enumeration/HealthStatusEnumResponse.java]: How many Enums that do the same thing do you wanna have

**Review by Maje-afk — CHANGES_REQUESTED** [review]

File: DiscordBot/DiscordBotCore/src/main/java/com/tainavbot/server/discordbot/domain/enumeration/HealthStatusEnum.java (hunk 1 above)
Comment (Kilometerr) [reply DiscordBot/DiscordBotCore/src/main/java/com/tainavbot/server/discordbot/domain/enumeration/HealthStatusEnum.java]: fixed

File: Orchestrator/src/main/java/com/tainavbot/server/orchestrator/domain/enumeration/HealthStatusEnum.java (hunk 2 above)
Comment (Kilometerr) [reply Orchestrator/src/main/java/com/tainavbot/server/orchestrator/domain/enumeration/HealthStatusEnum.java]: Needs to be enum per layer of hexagonal infrastructure

File: Orchestrator/src/main/java/com/tainavbot/server/orchestrator/infrastructure/adapter/primary/api/dto/enumeration/HealthStatusEnumResponse.java (hunk 3 above)
Comment (Kilometerr) [reply Orchestrator/src/main/java/com/tainavbot/server/orchestrator/infrastructure/adapter/primary/api/dto/enumeration/HealthStatusEnumResponse.java]: up and braught down to minimum in next version

Comment (Maje-afk) [reply Orchestrator/src/main/java/com/tainavbot/server/orchestrator/infrastructure/adapter/primary/api/dto/enumeration/HealthStatusEnumResponse.java]: *brought

**Pull request merged**